ui/overview_page.go

```diff
@@ -486,15 +563,13 @@ func (pg *overviewPage) syncBoxTitleRow(gtx layout.Context, inset layout.Inset)
 
 	return inset.Layout(gtx, func(gtx C) D {
 		return layout.Flex{Axis: layout.Horizontal}.Layout(gtx,
-			layout.Rigid(func(gtx C) D {
-				return statusTitleLabel.Layout(gtx)
-			}),
+			layout.Rigid(statusTitleLabel.Layout),
 			layout.Flexed(1, func(gtx C) D {
 				return layout.E.Layout(gtx, func(gtx C) D {
 					return layout.Flex{Axis: layout.Horizontal}.Layout(gtx,
 						layout.Rigid(func(gtx C) D {
-							return layout.Inset{Top: values.MarginPadding5, Right: values.MarginPadding20}.Layout(gtx, func(gtx C) D {
-								return pg.walletStatusIcon.Layout(gtx, values.MarginPadding10)
+							return layout.Inset{Top: values.MarginPadding4, Right: values.MarginPadding4}.Layout(gtx, func(gtx C) D {
+								return pg.walletStatusIcon.Layout(gtx, values.TextSize14)
```

Comment:
TextSize dimension shouldn't be used for margin or padding. Kindly define an equivalent of this dimension as MarginPadding